services/core-api/app/api/mines/reports/models/mine_report.py

```diff
@@ -1,4 +1,5 @@
 import uuid
+from datetime import datetime, timezone
```

Comment:
minor, but are these new imports being used?

Reply:
Ah, good catch.  I love cleaning up unused imports!  Otherwise this looks great Isuru!

Reply:
Removed unused imports :)